### PR TITLE
stop producing csv files in blast

### PR DESCRIPTION
**Pull request by hsiaoyi0504**
Replace #147.

### CODE REVIEW

**Comment by hsiaoyi0504** [comment]
Finished rebase for #151.

File: README.md [hunk 1]
```diff
@@ -20,7 +20,7 @@ Genomics-Workspace is a bioinformatic website project created by [i5k Workspace]
 * Task queue with [Celery](https://github.com/celery/celery) and [RabbitMQ](http://www.rabbitmq.com/).
 * Use [PostgreSQL](https://www.postgresql.org/) as database backend.
 * BLAST:
-  * generates all BLAST output formats for download: Text, CSV, TSV, XML, ASN.1.
+  * generates serveral BLAST output formats for download: Text, TSV, XML, ASN.1.
```
Comment (childers) [inline README.md]: typo: several instead of serveral

Comment (hsiaoyi0504) [reply README.md]: Thank you for finding this. I have fixed and update this pull request.

File: blast/templates/blast/results.html [hunk 2]
```diff
@@ -3,7 +3,6 @@
 {% load pipeline %}
 
 {% block navbar%}
-<li><a href="./{{task_id}}">Results</a></li>
```
Comment (childers) [inline blast/templates/blast/results.html]: What's this?

Comment (childers) [reply blast/templates/blast/results.html]: Does not appear to be visible in production. This is overwritten with the new navbar.

**Pull request merged**